### PR TITLE
Compare error

### PR DESCRIPTION
**Pull request by Winnie08L**

### CODE REVIEW

File: examples/example.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    shape=(2,),
+    resolution=(31,),
+    n_bands=3,
+    type="bloch",
+    temperature=150,
+    FitMethod="poly fit",
+    n_polynomial=0,
+)
+
+plot_noise_kernel(system, config)
```
Comment (Matt-Ord) [inline examples/example.py:23]: needs a rebase - I would also the error plot to kernel.py

File: reduced_state_caldeira_leggett/plot.py [hunk 2]
```diff
@@ -243,8 +278,131 @@ def plot_2d_111_potential(
...
+    ax.legend()
     fig.show()
+    input()
+
 
+def plot_isotropic_kernel_percentage_error(
+    system: PeriodicSystem,
+    config: SimulationConfig,
+    *,
+    to_compare: bool = False,
```
Comment (Matt-Ord) [inline reduced_state_caldeira_leggett/plot.py:362]: just make config1 optional here? and remove to_compare. Rename config1 to base_config, and default to fft with no trucation

**Review by Matt-Ord — CHANGES_REQUESTED** [review]

File: reduced_state_caldeira_leggett/plot.py (hunk 2 above)
Comment (Winnie08L) [reply reduced_state_caldeira_leggett/plot.py:362]: i will fix these review comments in modify_compare_error_func in the time_and_compare_error branch

**Comment by Matt-Ord** [comment]
Converted to draft for visibility, just click ready to review when it's ready for me

**Pull request closed without merging**